### PR TITLE
A11y pass over

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
A number of copy and aria attrs to improve a11y. In doing so, have improved the SEO of the site.

## QA
- On Ubuntu hit `Super + Alt + S` to enable Orca.
- Use the arrow keys to move through the site and tab between links
- Ensure they are well read out and you understand everywhere the links will take you
- Use WAVE (browser plugin) to ensure we have no a11y errors remaining.

### CODE REVIEW

File: templates/base_layout.html [hunk 1]
```diff
@@ -67,7 +67,7 @@
 
   {% include "partial/_navigation.html" %}
 
-  <div id="main">
+  <div id="main-content">
```
Comment (petermakowski) [inline templates/base_layout.html:70]: 👍 I assume this is to keep in sync with "skip links" button?

File: templates/partial/_navigation.html [hunk 2]
```diff
@@ -2,16 +2,16 @@
   <div class="p-navigation__row">
     <div class="p-navigation__banner">
       <div class="p-navigation__logo u-vertically-center">
-        <a class="p-navigation__link" href="/" style="color:white;">
+        <a class="p-navigation__link" href="/" style="color:white;" aria-label="Homepage link">
```
Comment (petermakowski) [inline templates/partial/_navigation.html:5]: I think that "link" in the label is redundant? Does Orca not announce that it's a link anyway? VoiceOver does.

File: static/sass/styles.scss [hunk 3]
```diff
@@ -9,11 +9,15 @@
 
 // Custom styles
 .p-navigation__banner {
-  background-color: $ubuntu-orange;
+  background-color: #D14715;
```
Comment (petermakowski) [inline static/sass/styles.scss:12]: Where does this value come from?

File: templates/index.html [hunk 4]
```diff
@@ -18,17 +18,27 @@ <h1>High-availability SQLite</h1>
 <section class="p-strip">
   <div class="row">
     <div class="col-8">
-      <p>Dqlite (“distributed SQLite”) extends SQLite across a cluster of machines, with automatic failover and high-availability to keep your application running. It uses <a href="https://github.com/canonical/raft">C-Raft</a>, an optimised Raft implementation in C, to gain high-performance transactional consensus and fault tolerance while preserving SQlite’s  outstanding efficiency and tiny footprint.</p>
+      <h2>Getting started</h2>
```
Comment (petermakowski) [inline templates/index.html:21]: awesome example of how thinking about accessibility helps all users 💥

File: templates/base_layout.html (hunk 1 above)
Comment (anthonydillon) [reply templates/base_layout.html:70]: Yeah, just had the wrong id

File: templates/partial/_navigation.html (hunk 2 above)
Comment (anthonydillon) [reply templates/partial/_navigation.html:5]: Good point will update

File: static/sass/styles.scss (hunk 3 above)
Comment (anthonydillon) [reply static/sass/styles.scss:12]: Discussing options with brand

**Review by petermakowski — APPROVED** [review]

**Pull request merged**